setcontext/setcontext.py

```diff
@@ -0,0 +1,406 @@
+
+import fire
+import functools
+import json
+import os
+import re
+import pathlib
+import subprocess
+import sys
+from termcolor import cprint
+
+
+#Add pytest
+#Build framework for  commandline tools.
+
+"""
+Global variable for local git repo.
+"""
+DEBUG = True
+PATH = pathlib.Path.home() / "git"
+
+
+class CONTEXT:
```

Comment:
Python3 has a convenient enumerated class for this purpose
https://docs.python.org/3.11/howto/enum.html